app/src/main/java/vn/mbm/phimp/me/MapImageActivity.java

```diff
@@ -0,0 +1,51 @@
+package vn.mbm.phimp.me;
+
```

Comment:
@Abhi2424shek I made the separate activity because the image couldn't be loaded into the image view, it gives error for memory overflow. I used glide library for it, suggested by google itself. Also there is no other activity to just view the image.